docs/src/catalyst_applications/nonlinear_solve.md

```diff
@@ -0,0 +1,99 @@
+# [Finding Steady States using NonlinearSolve.jl](@id nonlinear_solve)
+
+We have previously described how `ReactionSystem` steady states can be found through [homotopy continuation](@ref homotopy_continuation). Catalyst also supports the creation of `NonlinearProblems` corresponding to `ODEProblems` with all left-hand side derivatives set to 0. These can be solved using the variety of nonlinear equation-solving algorithms implemented by [NonlinearSolve.jl](https://github.com/SciML/NonlinearSolve.jl), with the solutions corresponding to system steady states. Generally, this approach is preferred when one only wishes to find *a single solution* (with homotopy continuation being preferred when one wishes to find *all solutions*). 
```

Comment:
Might be worth mentioning Nonlinearsolve is also the only approach when the system does not involve rational(?) functions in the species for rate laws (or is it just mass action and the Catalyst specific rate functions we have like mm that work with homotopy continuation?).

Reply:
Very good point.